lib/recipes/organic-chemistry/bake

```diff
@@ -50,8 +50,14 @@ recipe = Kitchen::BookRecipe.new(book_short_name: :organic_chemistry) do |doc|
 
     MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata,
                           klass: 'visualizing-chemistry')
+    MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata,
+                          klass: 'mechanism-problems')
+    MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata,
+                          klass: 'energy-reaction')
     MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata,
                           klass: 'additional-problems')
+    MoveExercisesToEOC.v1(chapter: chapter, metadata_source: metadata,
+                          klass: 'general-problems')
```

Comment:
I think we should we also bake exercises within these sections like here:

![image](https://user-images.githubusercontent.com/40228854/214854240-c43cd9f5-55b0-40c1-9f98-4cfa278661f3.png)

Reply:
Thank you for the catch! Taken care of